packages/next-ui/ActionCard/ActionCardListForm.tsx

```diff
@@ -36,6 +39,8 @@ export function ActionCardListForm<
     ...other
   } = props
   const RenderItem = render as React.FC<ActionCardItemRenderProps<ActionCardItemBase>>
+  const [show, setShow] = React.useState(false)
+  const maxItems = 4
```

Comment:
Might it be an idea to move this to the underlying ActionCardList instead of ActionCardListForm? Also, it should have `showMoreAfter={4}` prop or something which isn't configured by default.

Maybe even ActionCardLayout?

Reply:
I do like the simple display block / none strategy here, keeps it simple for now.